src/components/v2/V2Project/index.tsx

```diff
@@ -89,7 +92,11 @@ export default function V2Project({
             size={GUTTER_PX}
             style={{ width: '100%' }}
           >
-            {/* TODO volume chart */}
+            <VolumeChart
+              style={{ height: 240 }}
+              createdAt={createdAt}
+              projectId={projectId}
```

Comment:
I think `VolumeChart` needs to take into account `cv` (contracts version) here - the subgraph query needs to know about `projectId` _and_ `cv` to know what data to return.